utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -172,7 +172,7 @@ impl ThreadStats {
             self.write_buf_drained = Default::default();
 
             let mut stat: Vec<_> = self.stat.iter().collect();
-            stat.sort_by(|x, y| (*x).0.cmp(&(*y).0));
+            stat.sort_by(|x, y| (*x).0.cmp(&*y.0));
```

Comment:
I wonder if this can be just `.sort_by_key`?

Reply:
@matklad Thanks, I replaced it.